src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -385,6 +385,9 @@ trait Definitions extends api.StandardDefinitions {
     lazy val JavaRepeatedParamClass = specialPolyClass(tpnme.JAVA_REPEATED_PARAM_CLASS_NAME, COVARIANT)(tparam => arrayType(tparam.tpe))
     lazy val RepeatedParamClass     = specialPolyClass(tpnme.REPEATED_PARAM_CLASS_NAME, COVARIANT)(tparam => seqType(tparam.tpe))
 
+    // TODO : requires ABSTRACT | TRAIT | FINAL ????
+    lazy val AnyKindClass   = enterNewClass(ScalaPackageClass, tpnme.AnyKind, AnyTpe :: Nil, ABSTRACT | TRAIT) markAllCompleted
```

Comment:
The existence of the fictional class `scala.AnyKind` could be made conditional on the experimental flag to avoid source incompatibilities with programs already using that name.